chart/skywalking/README.md

```diff
@@ -80,9 +80,9 @@ The following table lists the configurable parameters of the Skywalking chart an
 | `ui.service.loadBalancerIP`                                  | Load Balancer IP address                                                                         | `nil`                                |
 | `ui.service.annotations`                                     | Kubernetes service annotations                                                                   | `{}`                                 |
 | `ui.service.loadBalancerSourceRanges`                        | Limit load balancer source IPs to list of CIDRs (where available))                               | `[]`                                 |
-| `esInit.nodeAffinity`                                        | ES init job node affinity policy                                                                 | `{}`                                 |
-| `esInit.nodeSelector`                                        | ES init job labels for master pod assignment                                                     | `{}`                                 |
-| `esInit.tolerations`                                         | ES init job tolerations                                                                          | `[]`                                 |
+| `oapInit.nodeAffinity`                                       | OAP init job node affinity policy                                                                | `{}`                                 |
+| `oapInit.nodeSelector`                                       | OAP init job labels for master pod assignment                                                    | `{}`                                 |
+| `oapInit.tolerations`                                        | OAP init job tolerations                                                                         | `[]`                                 |
```

Comment:
These look like breaking changes, we should add `breaking change` warning for this?

Reply:
> These look like breaking changes, we should add `breaking change` warning for this?

They are breaking change, I will have some more breaking changes soon, let me add them in the change log before 4.4.0